lib/tasks/images.js

```diff
@@ -66,8 +66,8 @@ module.exports = function(gulp, config) {
 						fileClone.path = path.join(name.dir, name.name + '-' + size + name.ext)
 						fileClone.imageTransform = {
 							width: size,
-							height: file.imageData.aspectRatio ? Math.ceil(size/file.imageData.aspectRatio) : null,
-							options: file.imageData.options || {},
```

Comment:
Maybe print some warning if user uses options that it's not working anymore?

Reply:
Good idea!